### DISCUSSION TITLE
Enhancement: Show used Tokens

### DISCUSSION THREAD

**Discussion by SphaeroX** [danny-avila/LibreChat#425] [Feature Requests & Suggestions]
### What features would you like to see added?

an option to view the used tokens on every call to the API if chatGPT API is used.

another advantage is that you could set the max tokens for a model, meaning you can have a "continue response" pop up when they are exceeded. Like on this discussion: https://github.com/danny-avila/chatgpt-clone/discussions/380

### More details

see picture

+ add under the settings modal an option to active/deactive it

### Which components are impacted by your request?

UI

### Pictures

![image](https://github.com/danny-avila/chatgpt-clone/assets/6785110/6676ca6f-2f90-4c6b-aa46-2b1b5ff61436)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Good ideas! I plan on doing these for both counts, and both require a PR to the library that is handling the API calls. It's on my list for this weekend

**Reply by Pushpit07** [reply]
hey @danny-avila, was this idea worked upon?

**Reply by Pushpit07** [reply]
@danny-avila bringing this up
